utils/hare-bootstrap

```diff
@@ -53,6 +50,29 @@ say() {
     echo -n "$(date '+%F %T'): $*"
 }
 
+ssh_safe() {
+    ssh -n $node $* ||
```

Comment:
where is this `node` argument coming from, this function is not taking a node argument?

Reply:
Changed the ssh funtion.